chainer/link.py

```diff
@@ -197,6 +197,15 @@ def __init__(self):
         finally:
             self._within_init_scope = old_flag
 
+    def __call__(self, *args, **kwargs):
+        try:
+            forward = self.forward
+        except AttributeError:
+            raise TypeError(
+                '{} object has neither \'Link.__call__\' method overridden'
+                'nor \'forward\' method defined.'.format(self))
```

Comment:
Missing space between "overridden" and "nor"...

Reply:
Oops! Thanks!